lib/reverse_words.rb

```diff
@@ -1,6 +1,33 @@
 # A method to reverse each word in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n), where n is the number of chanracters in the string
+# Space complexity: O(1)
+def swap(starting_index, ending_index, my_words)
+  for j in (starting_index..(starting_index + (ending_index - starting_index - 1) / 2))
+    temp = my_words[j]
+    my_words[j] = my_words[starting_index + ending_index - j]
```

Comment:
It is not immediately clear what value `my_words[starting_index + ending_index - j]` is. Think about ways you can simplify this. See the similar comment made on your string reverse PR.